ross/__init__.py

```diff
@@ -10,5 +10,6 @@
 from .rotor_assembly import *
 from .shaft_element import *
 from .utils import visualize_matrix
+from .defects import *
```

Comment:
Travis checks if imports are sorted by running [ISORT](https://timothycrosley.github.io/isort/).
Try running it and [BLACK](https://black.readthedocs.io/en/stable/the_black_code_style.html) to formmat the code lines with all files, so it won't raise any error again.